app/app.html

```diff
@@ -1,6 +1,6 @@
 <ion-menu [content]="content">
 
-    <ion-toolbar>
+    <ion-toolbar primary>
```

Comment:
Changes the style of the toolbar to have the background of the primary color from the theme (currently blue).